bin/v3c-x86-64-linux

```diff
@@ -4,5 +4,5 @@ BIN=$(cd $BIN && pwd)
 RT=$(cd $BIN/../rt/ && pwd)
 LIB=$(cd $BIN/../lib/ && pwd)
 V3C=${V3C:=$BIN/v3c}
-RT_FILES=$(echo $RT/x86-64-linux/*.v3 $RT/native/*.v3 $RT/gc/*.v3 $LIB/util/*.v3)
-exec $V3C -heap-size=200m -stack-size=2m -target=x86-64-linux -rt.sttables -rt.gc -rt.gctables -rt.files="$RT_FILES" "$@"
+RT_FILES=$(echo $RT/x86-64-linux/*.v3 $RT/native/*.v3 $RT/gc/*.v3 $LIB/net/x86-64-linux/*.v3)
```

Comment:
I think I'd prefer to keep the library files out of the default compile command.

Reply:
I've removed them yes.